Documentation/project-docs/api-review-process.md

```diff
@@ -1,51 +1,37 @@
+sponsors the issue. This is usually the area owner for which the API proposal or design change request was filed for.
 
-* **Pull is tagged as "Needs API Review"**. The project owner then marks the pull request as "Needs API Review".
+3. **Discussion**. The goal of the discussion is to help the assignee to make a
+decision whether we want to pursue the proposal or not. In this phase, the goal
+isn't necessarily to perform an in-depth review; rather, we want to make sure
+that the proposal is actionable, i.e. has a concrete design, a sketch of the
+APIs and some code samples that show how it should be used. If changes are necessary, the requester is encouraged to edit the issue description. This allows folks joining later to understand the most recent proposal. To avoid confusion, the requester should maintain a tiny change log, like a bolded "Updates:" followed by a bullet point list of the updates that were being made.
 
-* **API review**. Using the information in the pull request we'll create an APIX file that constitutes the API delta. The API review board meets multiple times a week to review all PRs that are tagged as needing an API review.
+4. **Owner makes decision**. When the owner believes enough information is available to make a decision, shell will update the issue accordingly:
 
-* **API review notes are being published**. After the review, we'll publish the notes in the [API Review repository](https://github.com/dotnet/apireviews). A good example is the [review of immutable collections](https://github.com/dotnet/apireviews/tree/master/2015-01-07-immutable).
+    * **Mark for review**. If the owner believes the proposal is actionable, she will label the issue with `api-ready-for-review`.
+    * **Close as not actionable**. In case the issue didn't get enough traction to be distilled into a concrete proposal, she will close the issue.
+    * **Close as won't fix**. Similarly, if proposal is taking the product in a direction we don't want to go, the issue might also get closed.
 
-* **Pull request is updated with the results of the API Review**. Once the API review is complete, the project owner uploads the notes and API HTML diff, including all comments. The project owner also updates the PR accordingly, with either a call to action to address some concerns or a good to go indicator.
+5. **API gets reviewed**. The group conducting the review is called *FXDC*, which stands for *framework design core*. In the review, we'll take notes and provide feedback. After the review, we'll publish the notes in the [API Review repository](https://github.com/dotnet/apireviews). A good example is the [review of immutable collections](https://github.com/dotnet/apireviews/tree/master/2015-01-07-immutable). Multiple outcomes are possible:
 
-* **Pull request is merged**. When there are no issues - or the issues were addressed by the contributor, the PR is merged.
+    * **Approved**. In this case the label `api-ready-for-review` is replaced
```

Comment:
Same as today, we accept contributions. Do you think we should track this explicitly on the item? I don't think we ever used 'accepting PRs'.